Rubberduck.Resources/RubberduckUI.resx

```diff
@@ -1291,4 +1291,19 @@ NOTE: Restart is required for the setting to take effect.</value>
   <data name="Settings" xml:space="preserve">
     <value>Settings</value>
   </data>
+  <data name="InspectionResults_FilterByError" xml:space="preserve">
```

Comment:
Should we be creating more elements? I'm pretty sure we already have those words somewhere else in the resx file...

Reply:
@bclothier  correct. `CodeInspectionSeverity_{level}` keys, in `InspectionsUI.resx`.